get_projects.py

```diff
@@ -0,0 +1,81 @@
+import asyncio
+import fire
+import json
+import os
+import requests
+
+from oauthlib.oauth2 import BackendApplicationClient
+from requests_oauthlib import OAuth2Session
+
+
+def get_projects(base_url, client_id, client_secret):
+    url = base_url + "/graphql"
+    access_token = get_access_token(base_url, client_id, client_secret)
+    operations = get_operations("get_projects.json")
+
+    team_id = operations["variables"]["input"]["filter"]["teamId"]
```

Comment:
so to use this, the user have to edit the get_projects.json file? @michael-jong-ds

Reply:
yes, because the filters for get project are too many to list in the cli

Reply:
i see, got it